pages/three.js

```diff
@@ -30,9 +30,8 @@ var ThreePreview = function() {
 
         scene = new THREE.Scene();
 
-        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1, 20000);
+        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1e-5, 1e10);
```

Comment:
Also -- now it seems that the model (specifically, [Boombox](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/BoomBox)) for Three is zoomed out _so_ far when initially loaded.  Did you really mean to change these values?